src/preferences/colorpaletteeditor.cpp

```diff
@@ -31,7 +31,12 @@ ColorPaletteEditor::ColorPaletteEditor(QWidget* parent, bool showHotcueNumbers)
             QSizePolicy(QSizePolicy::Expanding, QSizePolicy::Preferred));
     pColorButtonLayout->addWidget(pExpander);
 
-    m_pRemoveColorButton = new QPushButton("-", this);
+    QIcon removeIcon = QIcon::fromTheme("list-remove", QIcon());
+    if (!removeIcon.isNull()) {
+        m_pRemoveColorButton = new QPushButton(removeIcon, "", this);
```

Comment:
@Holzhaus can you use a debugger to check if this or line 38 is executing?

Reply:
38

Reply:
That's bizarre. Line 38 is identical to the line in master that was removed.

Reply:
yeah, I also have no idea what causes that but it happens every time.

Reply:
Maybe we can just keep the old code and if the remove icon is not null, then set icon and label on the button afterwards?

Reply:
I'll check if that segfaults as well.

Reply:
https://github.com/mixxxdj/mixxx/pull/2630#issuecomment-609466113